plugins/amdgpu/amdgpu_plugin.c

```diff
@@ -1195,7 +1214,6 @@ static int dump_events(int fd,
 int amdgpu_plugin_dump_file(int fd, int id)
 {
 	struct kfd_ioctl_criu_process_info_args info_args = {0};
-	struct kfd_ioctl_criu_dumper_args args = {0};
 	int ret;
```

Comment:
Since we are redoing the APIs so lets follow reverse-xmas notation for the variable declarations i.e. in decreasing order of length with each newline so this goes to the last line.

Reply:
Changed some other places, but info_args  is the longest line and already on top